.github/workflows/build-test-virtual.yml

```diff
@@ -36,4 +40,12 @@ jobs:
       - run: ./scripts/ci-checks.sh
       - run: ./build.sh
       - run: ./run_unit_tests.sh
-      - run: ./run_functional_tests.sh --enable-prefix-tree --enable-faketime
+      - run: ./run_functional_tests.sh --enable-faketime
+
+      - name: Upload logs
+        uses: actions/upload-artifact@v4
+        if: ${{ failure() }}
+        with:
+          name: out
```

Comment:
Is that right? Is self.workspace really out/? Even so, it makes sense to only upload the logs, the binaries will be large, and the upload will take a long time. See the CCF ci for a sample.

Reply:
It's only logs now, no one else uses it.

The idea of OB pipeline is that this's the directory you want to save after the run, so you only write there something you might need to investigate the failure.